lib/presentation/core/app_widget.dart

```diff
@@ -52,17 +29,9 @@ class _AppWidgetState extends State<AppWidget> {
         color: Colors.white,
         title: 'CollAction',
         theme: lightTheme(context),
-        // Do NOT hide banner in debug mode to distinguish between release and non-release builds!
-        // debugShowCheckedModeBanner: false,
```

Comment:
On that note... maybe we should create debug builds on development and release builds only on master?

Reply:
That would be ideal, let's add a task for this, it's how it should be done after all, and it's just adding a flag.